game-core/src/test/java/games/strategy/engine/lobby/client/login/LobbyPropertyFileParserTest.java

```diff
@@ -111,17 +115,32 @@ void checkVersionSelection() throws Exception {
   private static class TestProps {
     String host;
     String port;
+    URI httpHostUri;
     String message;
     String errorMessage;
     String version;
 
+
     String toYaml() {
-      final String printVersion = (version == null) ? "" : "  version: " + version + "\n";
-      return "- host: " + host + "\n"
-          + printVersion
-          + "  port: " + port + "\n"
-          + "  message: " + message + "\n"
-          + "  error_message: " + errorMessage + "\n";
+      final String printVersion = (version == null) ? "" : "version: \"" + version + "\"\n";
```

Comment:
It looks like the trailing newline is no longer necessary based on your new format string below.

Reply:
removed in: 9ab5fe9a7f19d74d3853010aecf8d8e2f7b40bb6